src/libtpm2-totp.c

```diff
@@ -27,10 +26,9 @@
 
 const TPM2B_DIGEST ownerauth = { .size = 0 };
 
-#define dbg(m, ...) fprintf(stderr, m "\n", ##__VA_ARGS__)
+#define dbg(m, ...)
```

Comment:
Why are we disabling the dbg outputs ?

Reply:
Fair point, I didn't want to clutter the output with additional output normal users do not need ("Calling Esys_GetRandom", "Calling Esys_CreatePrimary") or that is already covered by the new error messages ("Password required.", "No unseal blob included."). I agree that no-oping the macro is weird, so I now made it conditional on whether debugging is enabled (`./configure --enable-debug`) or not.